commercetools-models/src/main/java/io/sphere/sdk/shippingmethods/CartScore.java

```diff
@@ -0,0 +1,35 @@
+package io.sphere.sdk.shippingmethods;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.annotations.ResourceValue;
+import io.sphere.sdk.products.Price;
+
+import javax.annotation.Nullable;
+import javax.money.MonetaryAmount;
+
+@ResourceValue
+@JsonDeserialize(as = CartScoreImpl.class)
+public interface CartScore extends ShippingRatePriceTier {
+
+
+    @Override
+    default String getType(){
+        return "CartScore";
+    }
+
+    @Nullable
+    Long getScore();
+
+    @Nullable
+    PriceFunction getPriceFunction();
+
+    static CartScore ofPriceFunction(final PriceFunction priceFunction, final Long score){
```

Comment:
I'm fine with this factory method, but we could generate a builder class here too with our @HasBuilder annotation.